JWBestPracticeApps/FeedTableViewController/FeedTableViewController/FeedTableViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  FeedTableViewController.swift
+//  FeedTableViewController
+//
+//  Created by Amitai Blickstein on 6/26/22.
+//
+
+import UIKit
+
+class FeedTableViewController: UITableViewController {
+    private var viewModel = FeedViewModel.shared
```

Comment:
An excellent example of what I described in my FeedViewModel comment. Don't use a singleton. FeedViewModel should be instantiated here. i.e.` private var viewModel = FeedViewModel()`

Reply:
Incorporated.